sampler/src/AgateEnvironment/AgateEnvironment.js

```diff
@@ -129,22 +143,34 @@ const StorybookDecorator = (story, config) => {
 		}
 	};
 	const skinFromURL = getPropFromURL('skin');
-	const currentSkin = skinFromURL ? skinFromURL : 'gallium';
+	const currentSkin = skinFromURL ? skinFromURL : Config.defaultProps.skin;
 	const newSkin = (memory.skin !== currentSkin);
 	memory.skin = currentSkin;  // Remember the skin for the next time we load.
 	const accentFromURL = getPropFromURL('accent');
 	const highlightFromURL = getPropFromURL('highlight');
+	const allSkins = boolean('show all skins', Config);
```

Comment:
FWIW, props with spaces do not get rendered properly into a URL by storybook.  This is a bug in storybook (I don't know if it's been filed or not).  Should we update knob names to have a `-` or `_`?  (Would `%20` work?)

Reply:
See, that URL doesn't work.

Reply:
Of course it works. Copy and paste it, the whole thing, into a browser, or put it into a real `href` attribute and it will work just fine. Automatic URL parsers will obviously terminate on a space, but that doesn't mean the URL itself doesn't work.

Reply:
When I click copy with the all skins and night mode checked, I got this url `http://0.0.0.0:8080/?path=/story/agate--button&knob-animateOnRender=&knob-animationDelay=&knob-backgroundOpacity=opaque&knob-disabled=&knob-highlighted=&knob-selected=&knob-size=large&knob-type=standard&knob-children=Click me&knob-show all skins=true&knob-skin=gallium&knob-night mode=true&knob-accent=#8b7efe&knob-highlight=#e16253`